static/src/javascripts/projects/commercial/modules/messenger.ts

```diff
@@ -145,22 +145,24 @@ const toStandardMessage = (
  *
  * Incoming messages contain the ID of the iframe into which the source window is embedded.
  */
-const getIframe = (data: StandardMessage) => {
+const getIframe = (data: StandardMessage): HTMLIFrameElement | undefined => {
 	if (data.slotId) {
 		const container = document.getElementById(`dfp-ad--${data.slotId}`);
 		const iframes = container
 			? container.getElementsByTagName('iframe')
 			: null;
-		return iframes?.length ? iframes[0] : null;
+		return iframes?.length ? iframes[0] : undefined;
 	} else if (data.iframeId) {
 		const el = document.getElementById(data.iframeId);
```

Comment:
Maybe we can skip checking wether the element is an iframe by using a more specific selector?

```suggestion
		const el = document.querySelector(`iframe#${data.iframeId}`);
```